api/oc_udp.c

```diff
@@ -0,0 +1,100 @@
+/****************************************************************************
+ *
+ * Copyright (c) 2022 plgd.dev s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"),
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <assert.h>
+#include "oc_endpoint.h"
+#include "oc_udp_internal.h"
+#include "messaging/coap/coap.h"
+#include "port/oc_connectivity.h"
+#ifdef OC_SECURITY
+#include <mbedtls/ssl.h>
+#ifdef OC_OSCORE
+#include "messaging/coap/oscore.h"
+#endif /* OC_OSCORE */
+#endif /* OC_SECURITY */
+
+#include <assert.h>
```

Comment:
assert.h twice, my convention is internal headers in `""` first and then third-party headers in `<>`, so I would remove the include on line 19

Reply:
fixed